RepClient/src/app/shared/component/header/header.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+import { provideMockStore } from '@ngrx/store/testing';
+import { SharedModule } from '../../shared.module';
+
+import { HeaderComponent } from './header.component';
+
+describe('HeaderComponent', () => {
+  let component: HeaderComponent;
+  let fixture: ComponentFixture<HeaderComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [HeaderComponent],
+      imports: [
+        IonicModule.forRoot(),
+        SharedModule
+      ],
+      providers: [
+        provideMockStore()
+      ]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(HeaderComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
This test alone is not enough

Reply:
Tests where deemed not appropriate for this pull request and would be added as a separate pull request. However that was put on hold for now to aid with the Admin front end as it is more pressing to have it finished.